tests/testapps/CTCatalog/CTCatalog/Tests/CTCAlignmentTestViewController.m

```diff
@@ -150,8 +153,9 @@ - (void)viewDidAppear:(BOOL)animated {
 }
 
```

Comment:
You need it for the tests

---
In reply to: [79076875](https://github.com/Microsoft/WinObjC/pull/1010#discussion_r79076875) [](ancestors = 79076875)